src/chem/Residue.js

```diff
@@ -197,7 +197,8 @@ class Residue {
 
     const lp = getAtomPosition(this._leadAtom);
     const currLeadPos = new THREE.Vector3(lp.x, lp.y, lp.z);
-    if ((this._type.flags & ResidueType.Flags.NUCLEIC) !== 0) {
+    if (((this._type.flags & ResidueType.Flags.NUCLEIC) !== 0
+    || (this._type.flags & ResidueType.Flags.WATER) !== 0)) {
```

Comment:
Shouldn't there be an indentation? And what about other residue types/classes (e.g. added in the future)? Shouldn't we invert the condition or change it in some other way to make generic?

Reply:
I will think about it, thanks